ksql-functional-tests/src/test/resources/query-validation-tests/sink-partitions-replicas.json

```diff
@@ -10,7 +10,7 @@
       "topics": [
         {
           "name": "input",
-          "schema": {"name": "blah", "type": "record", "fields": [{"name": "c1", "type": "int"}]},
+          "schema": {"name": "blah", "type": "record", "fields": [{"name": "C1", "type": "int"}]},
```

Comment:
Our avro serde in KSQL is case insensitive and changes all names to upper case but the avro serializer in the testing tool is case sensitive. So to make them work correctly, I just used upper case so the column names will be compatible.

Reply:
Shouldn't we try to have the Avro serde used by the testing tool match the behavior of the one used by KSQL itself? Otherwise the testing tool won't be an accurate reflection of KSQL. I realize that in this case it's not too bad since we still maintain that anything that works with the testing tool works with KSQL, but it'd also be nice to have the property that anything that works with KSQL works with the testing tool as well.